src/builder.rs

```diff
@@ -57,21 +59,23 @@ impl From<value::OverflowError> for Error {
 }
 
 /// Information about a specific note to be spent in an [`Action`].
-#[derive(Debug)]
+#[derive(Debug, Clone)]
 struct SpendInfo {
     dummy_sk: Option<SpendingKey>,
     fvk: FullViewingKey,
     scope: Scope,
     note: Note,
     merkle_path: MerklePath,
+    // a flag to indicate whether the value of the note will be counted in the value sum of the action.
```

Comment:
Make sure that the comment reflects the sense of the flag. 
```suggestion
    // If this flag is true, the value of the note will not be counted in the value sum of the action.
```

Reply:
done